core/arch/riscv/mm/core_mmu_arch.c

```diff
@@ -720,8 +735,10 @@ void core_init_mmu_regs(struct core_mmu_config *cfg)
 {
 	struct mmu_partition *prtn = core_mmu_get_prtn();
 
-	cfg->satp = core_mmu_pgt_to_satp(prtn->asid,
-					 core_mmu_get_root_pgt_va(prtn));
+	for (int core = 0; core < CFG_TEE_CORE_NB_CORE; core++) {
+		cfg->satp[core] = core_mmu_pgt_to_satp(prtn->asid,
+						       &prtn->root_pgt[core]);
+	}
```

Comment:
Define `core` at block entry and remove braces as per coding style.
Suggestion:
```c
	struct mmu_partition *p = core_mmu_get_prtn();
	unsigned int n = 0;

	for (n = 0; n < CFG_TEE_CORE_NB_CORE; n++)
		cfg->satp[n] = core_mmu_pgt_to_satp(p->asid, p->root_pgt + n);

Reply:
Changed in https://github.com/OP-TEE/optee_os/pull/6713/commits/22f2fd2636e42d119e0fdd39a61ccf677b4fc73c